yaksh/static/yaksh/js/add_question.js

```diff
@@ -76,8 +76,9 @@ function textareaformat()
 	document.getElementById('id_type').setAttribute('class','select-type');
 	document.getElementById('id_points').setAttribute('class','mini-text');
     document.getElementById('id_tags').setAttribute('class','tag-text');
-    
-	
+    $("[id*="+'test_case_args'+"]").attr('placeholder',
```

Comment:
I felt adding help text directly in Testcase models may not be a good idea, so I have added this js part to add help text in Testcase Forms in template.